lib/elixir/lib/uri.ex

```diff
@@ -375,15 +375,19 @@ defmodule URI do
   defdelegate to_string(uri), to: String.Chars.URI
 
   @doc ~S"""
-  Merges two URIs as per RFC 3986 §5.2
+  Merges two URIs.
+
+  This function merges two URIs as per [RFC
```

Comment:
I n other places (e.g. https://github.com/whatyouhide/elixir/blob/8e2e2262557f3dceabef9f01e920a929fc694d9a/lib/elixir/lib/uri.ex#L164) we different wording and style. :bowtie: